examples/rl/ddpg_pendulum.py

```diff
@@ -241,16 +236,72 @@ def update_target(tau):
 """
 
 
-def get_actor():
-    # Initialize weights between -3e-3 and 3-e3
-    last_init = tf.random_uniform_initializer(minval=-0.003, maxval=0.003)
+class Actor(tf.keras.Model):
```

Comment:
Please stick to the functional API definition for the actor and critic models

Reply:
I am not sure how to fix this. I thought creating model classes would still adhere to the functional API. The functions get_actor() and get_critic() are still contained within the file, but the forward pass code has been moved to the "call" methods within their respective classes.

Reply:
Or do you mean that the current definition should not be changed at all?

Reply:
Fixed! Changing the model definition was indeed unnecessary.